e2e/test/scenarios/visualizations/table.cy.spec.js

```diff
@@ -307,6 +314,71 @@ describe("scenarios > visualizations > table", () => {
   });
 });
 
+describe("scenarios > visualizations > table > conditional formatting", () => {
+  beforeEach(() => {
+    resetTestTable({ type: "postgres", table: "many_data_types" });
+    restore(`postgres-writable`);
+    cy.signInAsAdmin();
+    resyncDatabase({
+      dbId: WRITABLE_DB_ID,
+      tableName: "many_data_types",
+    });
+
+    getTable({ name: "many_data_types" }).then(({ id: tableId, fields }) => {
+      const booleanField = fields.find(field => field.name === "boolean");
+      const stringField = fields.find(field => field.name === "string");
+      const idField = fields.find(field => field.name === "id");
+
+      visitQuestionAdhoc({
+        dataset_query: {
+          database: WRITABLE_DB_ID,
+          query: {
+            "source-table": tableId,
+            fields: [
+              ["field", idField.id, { "base-type": idField["base_type"] }],
+              [
+                "field",
+                stringField.id,
+                { "base-type": stringField["base_type"] },
+              ],
+              [
+                "field",
+                booleanField.id,
+                { "base-type": booleanField["base_type"] },
+              ],
+            ],
+          },
+          type: "query",
+        },
+        display: "table",
+      });
+    });
+  });
+
+  it("should work with boolean columns", () => {
+    cy.findByTestId("viz-settings-button").click();
+    leftSidebar().findByText("Conditional Formatting").click();
+    cy.findByRole("button", { name: /add a rule/i }).click();
+
+    popover().findByRole("option", { name: "Boolean" }).click();
+
+    //Dismiss popover
+    leftSidebar().findByText("Which columns should be affected?").click();
+
+    //Check that is-true was applied by default to boolean field rule
+    cy.findByTestId("conditional-formatting-value-operator-button").should(
+      "contain.text",
+      "is true",
+    );
+
+    cy.findByRole("gridcell", { name: "true" }).should(
+      "have.css",
+      "background-color",
+      "rgba(80, 158, 227, 0.65)",
```

Comment:
Can we change this since it is not the specific color that is important, just that the brand color is being used?
```suggestion
      color("brand"),
```